scripts/generator/graphql_generator/csharp/SDK/UnityLoader.cs.erb

```diff
@@ -78,5 +78,11 @@ namespace Shopify.Unity.SDK {
             return "unity";
         }
     }
+
+    public class UnityLoaderProvider : ILoaderProvider {
+        BaseLoader ILoaderProvider.GetLoader(string accessToken, string domain) {
```

Comment:
I haven't seen this before - prefixing the method name with the interface type. What's the reason for doing this vs just implementing `GetLoader`?

Reply:
Its called explicit interface implementation, the neat thing about it is that the explicit interface methods are only available when the class is cast to the interface type, which ensures that we are using the least specific reference to the type. It also has the added niceness of knowing what methods are satisfying a particular interface at a glance.